src/Http/TransformsFlexibleErrors.php

```diff
@@ -91,6 +91,10 @@ protected function transformMessages($messages, $key, $attribute)
         $search = str_replace('_', ' ', Str::snake($key));
         $attribute = str_replace('_', ' ', Str::snake($attribute->name));
 
+        if(!Str::startsWith($translated = trans('validation.attributes.'.$attribute), 'validation.attributes.')) {
```

Comment:
Although I'm sure it works, this is a pretty odd way to check for the existence of the translation imo. Is there a specific reason you went for this instead of `Lang::hasForLocale('validation.attributes.'.$attribute', app()->getLocale())`?

Reply:
Sorry, maybe even `Lang::has('validation.attributes.'.$attribute')` actually. It should take care of the current locale automatically, my bad.

Reply:
Yes you're right, thanks for that feedback. I just pushed another commit.